barretenberg/cpp/src/barretenberg/stdlib/primitives/field/field.cpp

```diff
@@ -388,6 +388,19 @@ template <typename Builder> field_t<Builder> field_t<Builder>::pow(const field_t
     return accumulator;
 }
 
+/**
+ * @brief raise a field_t to a power of an exponent (field_t). Note that the exponent must not exceed 32 bits and is
+ * implicitly range constrained.
+ *
+ * @returns this ** (exponent)
+ */
+template <typename Builder> field_t<Builder> field_t<Builder>::pow(const size_t exponent) const
```

Comment:
leave a TODO here? This probably shouldn't exist and the other one shouldn't allow conversions from size_t to field_t perhaps.

Reply:
ah nvm I see the below